src/Tests/Libs/GirTest-0.1.Tests/PropertyTest.cs

```diff
@@ -100,6 +100,16 @@ public void TestBooleanProperty(bool b)
         obj.BooleanValue.Should().Be(b);
     }
 
+    [DataTestMethod]
+    [DataRow(42ul)]
```

Comment:
Can you add [Uint64.MaxValue](https://learn.microsoft.com/en-us/dotnet/api/system.uint64.maxvalue?view=net-9.0) and Uint64.MinValue to the tests?

Reply:
Sure, I've added that in now 👍